docs/gettsim_objects/input_variables.md

```diff
@@ -451,10 +451,26 @@ Type: float
 
 ## `y_pflichtbeitr_ab_40`
 
-Total years of mandat. contributions afterage 40
+Total years of mandat. contributions after age 40
 
 Type: float
 
+(pflichtbeitr_8_in_10)=
+
+## `pflichtbeitr_8_in_10`
+
+Has at least 8 contribution years in past 10 years
+
+Type: bool
+
+(arbeitsl_1y_past_585)=
+
+## `arbeitsl_1y_past_585`
```

Comment:
Since we are using `m_arbeitslos` for the same concept, we should make that consistent. I.e., eihter

- `m_arbeitslos` and `arbeitslos_1y_past_585` 

or

- `m_arbeitsl` and `arbeitsl_1y_past_585` 

 Sorry for being so pedantic, but else this whole thing will blow up.

And same then for `_ges_rente_arbeitslos_altersgrenze` and `_ges_rente_vorauss_arbeitslos`

Reply:
done